inspirehep/config.py

```diff
@@ -1095,6 +1095,28 @@ def _(x):
 # Inspire mappings
 # ================
 
+INSPIRE_LEGACY_ROLES = {
+    'editing': [
+        'ed.',
+        'eds.',
+        'Ed.',
+        'ed,,',
+        'eds',
+        'ed,',
+        'ed. et al.',
+        'et al.'
```

Comment:
I believe `et al.` shouldn't be there. It's due to when people are cut & pasting from a reference.